modules/imaging_browser/test/TestPlan.md

```diff
@@ -1,44 +1,43 @@
 ## Imaging Browser test plan
 	
-### A. Imaging Browser front page
-1. User can load Imaging Browser module front page IFF has permission imaging_browser_view_site, imaging_browser_view_allsites, imaging_browser_phantom_allsites or imaging_browser_phantom_ownsite [Partial Automation Testing; To ADD phantom permissions automated tests]
```

Comment:
@cmadjar I removed the `To ADD phantom permissions automated tests` I think these should be issues not in the test plan.

If you agree, I'll create the issues

Reply:
totally agree! 

However, as long as there are no automated testing for that point, I would keep it in the test plan since it still needs to be tested in the meantime, no? I would just remove the part in [] from the test plan, create an issue and then mark in the issue that the test plan need to be modified once the automation is done so we do not forget to update the test plan.

Does that make sense?

Reply:
makes sens, do you think you can formulate the item for the test plan ? not sure how to formulate what should be tested for phantoms

Reply:
what is wrong with the way it is written? A user would see the imaging browser module if he/she has at least one of those permissions. It does not really matter which one I think.

Reply:
got it